### ISSUE TITLE
CookieError("Invalid attribute %r" % (K,))

### ISSUE THREAD

**Issue by diedrichg** [alandtse/alexa_media_player#2508]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/alandtse/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**

Logger: homeassistant.config_entries
Source: config_entries.py:604
First occurred: 12:28:05 PM (1 occurrences)
Last logged: 12:28:05 PM

Error setting up entry diedrichguenther@gmail.com - amazon.com for alexa_media
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 604, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/alexa_media/__init__.py", line 361, in async_setup_entry
    await login.login(cookies=await login.load_cookie())
                              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/alexapy/alexalogin.py", line 328, in load_cookie
    cookies = pickle.loads(await myfile.read())
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/cookies.py", line 312, in __setitem__
    raise CookieError("Invalid attribute %r" % (K,))
http.cookies.CookieError: Invalid attribute 'partitioned'

**To Reproduce**

**Expected behavior**

**Screenshots**

**System details**

- Home Assistant version: 2024.9
- alexa_media version (from `const.py` or HA startup log):
- alexapy version (from `pip show alexapy` in homeasssistant container or HA startup log): 4.12.12
- Is Amazon [2FA/2SV enabled](https://github.com/alandtse/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account)  (y/n): n
- Amazon Domain:

**Debug Logs (alexa_media & alexapy)**
Please provide [logs](https://github.com/alandtse/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).

**Additional context**

**Comment by ruipalhinhas** [comment]
I'm getting this error as well, it seems to only trigger at Home Assistant start up/every restart. I then need to go in and manually reload the integration, it then works fine.

Not sure if this is related to the blocking calls being flagged or potentially blocked by HA since the September update (2024.9).

```
Logger: homeassistant.config_entries
Source: config_entries.py:604
First occurred: 15:56:59 (1 occurrences)
Last logged: 15:56:59

Error setting up entry xxx@xxx - amazon.co.uk for alexa_media
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 604, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/alexa_media/__init__.py", line 361, in async_setup_entry
    await login.login(cookies=await login.load_cookie())
                              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/alexapy/alexalogin.py", line 328, in load_cookie
    cookies = pickle.loads(await myfile.read())
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/cookies.py", line 312, in __setitem__
    raise CookieError("Invalid attribute %r" % (K,))
http.cookies.CookieError: Invalid attribute 'partitioned'
```

**Comment by danielbrunt57** [comment]
Duplicate of #2288 
Resolved by alexapy v1.29.2:
```python
"""Ensure cookies.Morsel contains "partitioned"
   See: https://github.com/python/cpython/issues/112713
"""
partitioned = { "partitioned" : "Partitioned" }
Morsel._reserved.update(partitioned)
Morsel._flags.add("partitioned")
_LOGGER.debug("http.cookies patch: Morsel._reserved: %s; Morsel._flags: %s", partitioned, Morsel._flags)
```
